pkg/sdk/secrets_def.go

```diff
@@ -241,6 +241,8 @@ var SecretsDef = g.NewInterface(
 		SQL("SECRETS").
 		OptionalLike().
 		OptionalExtendedIn(),
+	g.ResourceIDHelperMethod,
+	g.ResourceObjectTypeHelperMethod,
```

Comment:
Similarly to the comment above: `g.ShowObjectTypeMethod`?

Reply:
seems good